### ISSUE TITLE
addToDictionary fails in spellcheck plugin

### ISSUE THREAD

**Issue by peterlsmith** [tinymce/tinymce#4213]
Suspected Defect in SpellCheck plugin, tinymce version 4.7.6 dev package.

The SpellCheck plugin generates a javascript error when using the 'addToDictionary' option from the popup menu. i.e. spell check a page, move the cursor over a mis-spelled word, right click to popup the dictionary menu, then select 'Add to Dictionary' option.

In Firefox 50.0.2 this is reported as:
TypeError: currentLanguageState.get is not a function[Learn More]

at line 389 in plugin.js. No http request is submitted to the configured dictionary server back-end. 

The appears to  be a missing parameter in the 'sendRpcCall' call at line 453 in plugin.js. The 'addToDictionary' argument should be preceded by 'startedState'. Adding in the missing parameter allows the function to work as expected (sending an http request to the dictionary server).

```
  var addToDictionary = function (editor, pluginUrl, startedState, textMatcherState, word, spans) {
    editor.setProgressState(true);
    **sendRpcCall(editor, pluginUrl, 'addToDictionary', word, function ()** {
      editor.setProgressState(false);
      editor.dom.remove(spans, true);
      checkIfFinished(editor, startedState, textMatcherState);
    }, function (message) {
      editor.notificationManager.open({
        text: message,
        type: 'error'
      });
      editor.setProgressState(false);
    });
  };

```

**Comment by peterlsmith** [comment]
'startedState' is not the right value to pass through (unless your chosen language is called 'true') - it needs 'currentLanguageState', but I'm not sure how to get that.

**Comment by fyrkant** [comment]
This has been fixed in 4.7.7 released today.

Closing this issue but if you can reproduce it in 4.7.7 leave a comment and I can open it up again.
